### DISCUSSION TITLE
忘记了锁屏密码怎么办

### DISCUSSION THREAD

**Discussion by yxzwym** [kingToolbox/WindTerm#281] [Q&A] [answered]
如何重置锁屏密码

**Comment by kingToolbox** [comment, marked as answer]
If you want to reset the master password, please open `WindTerm_install_path/profiles/default.v10/user.config` and remove the configuration items `application.fingerprint` and `application.masterPassword`. Note that all automatic login information encrypted with the old master password will be lost.

**Reply by sammytan** [reply]
Reset master password fail used by WindTerm 2.4.0 on macOS monterey 12.3.1 and collapse when exit WindTerm . Pls fix ,Thanks
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/17334441/163954620-a3e27b69-ed03-4a86-a8f1-5da7b97b0cba.png">
<img width="1443" alt="image" src="https://user-images.githubusercontent.com/17334441/163955088-195717c8-627c-4c1c-9a1c-87c8e1502a20.png">
Log detail as https://github.com/kingToolbox/WindTerm/issues/672

@kingToolbox

**Reply by kingToolbox** [reply]
Thank you, I have received multiple crash reports for the MacOS version and I will fix it in the next version.

As for the error `Change the master password failed`, I have not been able to reproduce the bug. You can try to restart `WindTerm` and change the master password again to see if the action is successful.

**Reply by kingToolbox** [reply]
@sammytan 
I am happy to inform you that [WindTerm_2.4.1](https://github.com/kingToolbox/WindTerm/releases/tag/2.4.0) has been released and the  crash has been fixed. Please download and check it, thank you.
